tests/crash/kill.py

```diff
@@ -0,0 +1,54 @@
+from functools import reduce
+import logging
+import time
+from typing import Callable
+
+import psutil
+
+TIMEOUT = 30
+AND_FILTERS = ["postgres", "vectors"]
+logging.getLogger().setLevel(logging.INFO)
+
+
+def process_filter(p: psutil.Process):
+    cmdline = "".join(p.cmdline())
+    filter_func: Callable[[bool, str], bool] = lambda ans, e: ans and (e in cmdline)
+    return reduce(filter_func, AND_FILTERS, True)
```

Comment:
This is a bad usage of `reduce`. Could you try the normal code?

Reply:
Fixed.